packages/libs/lambda-at-edge/src/lib/cleanRequestUriForRouter.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
I've yet to touch the response handler in by refactoring precisely to avoid conflicting with this pull request. Much of the remaining response handler routing logic can be refactored away, I believe. But it will be easier after both this and my currently open PR are merged to master.

Reply:
Cool, let's update this in a follow-up PR.